app/routes/notifications.js

```diff
@@ -2,41 +2,64 @@
 
 const express = require('express');
 const router = express.Router(); // eslint-disable-line new-cap
+const helpers = require('../../lib/helpers');
 const mobilecommons = rootRequire('lib/mobilecommons');
 const logger = app.locals.logger;
 const stathat = app.locals.stathat;
 
-router.post('/reminder', (req, res) => {
-  const northstarUserId = req.body.northstarUserId;
-  const campaignId = req.body.campaignId;
-  const reminderType = req.body.reminderType;
+const allowedReminderTypes = ['signup', 'reportback'];
 
-  if (!northstarUserId || !campaignId || !reminderType) {
-    res.json({ error: 'Missing parameters' });
-    return;
+router.post('/reminder', (req, res) => {
+  // Check required parameters.
+  if (!req.body.mobile) {
+    return helpers.sendResponse(res, 422, 'Missing required mobile parameter.');
+  }
+  if (!req.body.campaign_id) {
+    return helpers.sendResponse(res, 422, 'Missing required campaign id parameter.');
+  }
+  if (!req.body.reminder_type) {
+    return helpers.sendResponse(res, 422, 'Missing required reminder type parameter.');
   }
 
-  if (!app.locals.campaigns[campaignId]) {
-    res.json({ error: 'No campaign set in Gambit for the given campaign id.' });
-    return;
+  const mobile = req.body.mobile;
```

Comment:
I think it would be simpler to just define all of these variables first, then check if they are defined

```js
const mobile = req.body.mobile;

if (!mobile) { ... }
```

otherwise you're typing out `req.body.<something>` twice